### PR TITLE
ramips: add support for Rostelecom RT-FE-1A

### PR DESCRIPTION
**Pull request by csharper2005**
Rostelecom RT-FE-1A is a wireless WiFi 5 router manufactured by Sercomm company.

Device specification
--------------------
```
SoC Type: MediaTek MT7621AT
RAM: 256 MiB
Flash: 128 MiB
Wireless 2.4 GHz (MT7603EN): b/g/n, 2x2
Wireless 5 GHz (MT7615E): a/n/ac, 4x4
Ethernet: 5x GbE (WAN, LAN1, LAN2, LAN3, LAN4)
USB ports: No
Button: 2 buttons (Reset & WPS)
LEDs:
   - 1x Power (green, unmanaged)
   - 1x Status (green, gpio)
   - 1x 2.4G (green, hardware, mt76-phy0)
   - 1x 2.4G (blue, gpio)
   - 1x 5G (green, hardware, mt76-phy1)
   - 1x 5G (blue, gpio)
   - 5x Ethernet (green, hardware, 4x LAN & WAN) Power: 12 VDC, 1.5 A
Connector type: barrel
Bootloader: U-Boot
```

### CODE REVIEW

File: target/linux/ramips/dts/mt7621_rostelecom_rt-fe-1a.dts [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+		port@4 {
+			status = "okay";
+			label = "lan4";
+		};
+	};
+};
+
+&uartlite3 {
+	status = "okay";
+	current-speed = <57600>;
```
Comment (DragonBluep) [inline target/linux/ramips/dts/mt7621_rostelecom_rt-fe-1a.dts:264]: `uart3` pin group has already been used as `gpio` function?

Comment (csharper2005) [reply target/linux/ramips/dts/mt7621_rostelecom_rt-fe-1a.dts]: I mistakenly copy'n'pasted `uartlite3` block from another dts. Thanks. Fixed.

**Comment by csharper2005** [comment]
https://github.com/openwrt/openwrt/commit/6602cafe83095740de45543fad66ebbf06e40490:
- drop unnecessary `uartlite3`

**Review by raenye — APPROVED** [review]
LGTM.

**Review by neheb — APPROVED** [review]

**Comment by csharper2005** [comment]
https://github.com/openwrt/openwrt/commit/69d194c9393d0f6947770498d20acf837512077c:
- rebase
- remove led label due to https://github.com/openwrt/openwrt/pull/13837 merging

**Pull request merged**